gobblin-runtime/src/main/java/gobblin/runtime/MultiConverter.java

```diff
@@ -80,87 +79,122 @@ public Object convertSchema(Object inputSchema, WorkUnitState workUnit)
   }
 
   /**
-   * A type of {@link java.util.Iterator} to be used with {@link MultiConverter}.
+   * A type of {@link java.util.Iterator} to be used with {@link MultiConverter}. The Converter uses the
+   * {@link ChainedConverterIterator} to chain iterators together. The first ChainedConverterIterator created contains
```

Comment:
Use `{@link ChainedConverterIterator}` for all references to `ChainedConverterIterator `.